### PR TITLE
read the docs setup, plus fix broken links in README.rst

### LINKED ISSUE #64: Update completely outdated README.rst
Should provide a brief overview, then point to user stories, design documents, and other docs. Currently the README reflects team's decisions from a couple of months back, and it is very much out of date.

### PR DESCRIPTION
**Pull request by jimbaker**
Let's not have broken links in `README.rst` - once we actually move to OpenStack infra, we can atomically update this in one commit against master with a merged PR.

Because we now have some docs, it's useful to build transitionally with readthedocs.org, which is enabled here as well. This can be seen in this build: http://craton.readthedocs.io/en/craton-read-the-docs-setup/ Currently http://craton.readthedocs.org points at this build, however, we can update to master once this PR is merged.

Closes #64

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,36 +1,29 @@
...
-Project Resources
------------------
+Support for workflows, CLI, and the Python client is in progress.
 
-* Free software: Apache license, Version 2.0
+For more information, please refer to the following project resources:
 
-..
+* **Free software:** under the `Apache license <http://www.apache.org/licenses/LICENSE-2.0>`_
+* **Documentation:** http://craton.readthedocs.org
```
Comment (ChristopherMSpencer) [inline README.rst]: Should we add TODOs to update these links later?

**Comment by ChristopherMSpencer** [comment]
+1 lgtm

File: README.rst (hunk 1 above)
Comment (jimbaker) [inline README.rst]: I believe this answered by line 22, although perhaps this should be explicit. Making a note of that.

File: README.rst [hunk 2]
```diff
@@ -1,36 +1,29 @@
...
-
-* Source: https://git.openstack.org/cgit/openstack/craton
-
-* Bugs: http://bugs.launchpad.net/craton
-
-* Wiki: https://wiki.openstack.org/wiki/Craton
-
-Project status, bugs, and features are tracked on Launchpad:
- http://launchpad.net/craton
+We plan to move these links to OpenStack infrastructure once Craton
```
Comment (jimbaker) [inline README.rst]: Mark with a TODO to make explicit/easy to find

File: README.rst (hunk 1 above)
Comment (jimbaker) [inline README.rst]: Now explicit.

Comment (ChristopherMSpencer) [inline README.rst]: +1

**Pull request merged**